homework/react_2025/react-1-hw-1/components/ui/navBar/components/NavItem.jsx

```diff
@@ -0,0 +1,17 @@
+// import Link from "next/link";
+// import classNames from "classnames";
+// import styles from "./NavItem.module.css";
+
+// export function NavItem({ title, link, isActive, number }) {
+//   return (
+//     <li
+//       className={classNames(styles.navbarLinks, {
+//         [styles.isLinkActive]: isActive,
+//       })}
+//     >
+//       <Link href={link}>
+//         <b>{number}</b> {title}
+//       </Link>
+//     </li>
+//   );
+// }
```

Comment:
you have a good start here, but its not 100% there.

a few clues;

- watch the export statement on line 5. this is a .jsx file the export/import should follow convention.
- use Link from next/link. you have done a good job with those on the FooterNavLink component